src/UI/Component/Chart/Pick.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Component\Chart;
+
+/**
+ * Interface Pick Charts
+ */
+interface Pick {
+
+	/**
+	 * Sets a key value pair as items for the list. Key is used as title and value is a boolean marking highlighted values.
```

Comment:
Actually it is in my view correct. Did you comment the right line? But as pointed out before, we might change the passing data as array approach.